.config/brix/rust/search_test.txt

```diff
@@ -0,0 +1,3 @@
+test test test test wow most mest jester
```

Comment:
not really about this file, but don't forget to test the home directory, and if you specify a directory on the cli. For the cli you can probably just actually pass the path directly to your config object, and use a tmpdir. for home you might just find a way to set that to a tempdir for the test (not sure)